assets/dev/js/frontend/utils/anchors.js

```diff
@@ -1,3 +1,4 @@
+// TODO: Remove this file in a future version of Elementor.
```

Comment:
@rami-elementor 
Can we delete this file now already? Or should we wait for a future version?